BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import Shared
+
+struct AddCustomAssetView: View {
+  
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  @State private var nameInput = ""
+  @State private var addressInput = ""
+  @State private var symbolInput = ""
+  @State private var decimalsInput = ""
+  @State private var showError = false
+  
+  var userAssetStore: UserAssetsStore
+  
+  var body: some View {
+    NavigationView {
```

Comment:
Please check this on iPad, it may need `.navigationStyle(.stack)`

Reply:
Looks fine on iPad. I added new screenshot. Tested on iOS 14 and 15